democracy/locale/fi/LC_MESSAGES/django.po

```diff
@@ -196,6 +208,9 @@ msgstr "osat"
 msgid "service map URL"
 msgstr "palvelukartta-URL"
 
+msgid "slug"
+msgstr "slug"
```

Comment:
for real? :D well, actually was told to use "slugi" there, but apparently misspelled even that